src/delegation/DelegationRootNode.ts

```diff
@@ -48,10 +50,11 @@ export default class DelegationRootNode extends DelegationBaseNode
     return Promise.resolve(this)
   }
 
-  // tslint:disable-next-line:prefer-function-over-method
+  /* eslint-disable class-methods-use-this */
```

Comment:
Why do we need to disable the rule instead of making the method static?

Reply:
I actually just wanted to move the method to the BaseNode, but there is a problem with circular dependencies. I opened a ticket to resolve this issue: https://github.com/KILTprotocol/sdk-js/issues/144

Reply:
Moved to https://github.com/KILTprotocol/ticket/issues/4